LiteNetLib/Utils/NetDataWriter.cs

```diff
@@ -329,50 +336,30 @@ public void Put(IPEndPoint endPoint)
             Put(endPoint.Port);
         }
 
-        public void Put(string value)
```

Comment:
Don't delete public methods. This will cause `MissingMethodException` at runtime if assembly was not recompiled with a newer version of the library.